.github/release-please/manifest.json

```diff
@@ -1 +1 @@
-{ ".": "7.0.0" }
```

Comment:
This may cause a prettier war, as release please uses `JSON.stringify` :thinking:

Reply:
Ugh :/

Reply:
Made a PR to opt that file out of prettier: https://github.com/eslint-community/eslint-plugin-promise/pull/531